audio/templates/html/audio.html

```diff
@@ -19,7 +19,10 @@
       {% endif %}
     </audio>
     {% if allow_audio_download and audio_download_url %}
-      <div class="audio-xblock-download"><a href="{{ audio_download_url }}" target="_blank" rel="noopener" download>Download audio</a></div>
+      <p class="audio-xblock-download">
+          <a href="{{ audio_download_url }}" target="_blank" rel="noopener" download>Download audio</a>
+          (Some browsers may open this in a new tab instead of downloading. In this case, right click or long press, and select the "Save as" option.)
```

Comment:
```suggestion
          (Right click or long press on the download link and select the "Save as" option to download. Clicking on the link might open it for playback in a tab.)
```

Let's give them the advice that will almost always work in all browsers first, then give the reason

Reply:
Thanks @xitij2000 , added. :)